ngraph/core/reference/src/runtime/reference/reorg_yolo.cpp

```diff
@@ -76,9 +76,9 @@ namespace ngraph
                                 arg_index *= elem_size;
                                 dest_index *= elem_size;
 
-                                std::copy(arg + arg_index,
```

Comment:
Is the original implementation using decrease=false? If it is why it does not generate the same output as plaidml reorg yolo with decrease=false?

Reply:
the `decrease` mode is not just relative with the final "copy" (the order of `arg_index` and `dest_index` ), but also be relative with the ` imp_out_c, imp_out_h, imp_out_w`. 
For ngraph implement , it always turn output shape to` N, C/(stride*stride), H*stride, W*stride`, that means it only implement `decrease=True`, but with wrong "copy".

so,  the right condition is something like below:

`decrease=true` ---> output shape `N, C/(stride*stride), H*stride, W*stride`  ---->  arrayOut[**out_index**] = arrayIn[**in_index**]      

`decrease=false` ---> output shape `N, C*(stride*stride), H/stride , W/stride` ---->  arrayOut[**in_index**] = arrayIn[**out_index**] 

but our ngraph implement is like below : 

_`decrease=true` ---> output shape `N, C/(stride*stride), H*stride, W*stride` -----> arrayOut[**in_index**] = arrayIn[**out_index**]_ 

it means, whether we use `decrease=true` or `decrease=false`, it will generate the different result from our plaidml reorg_yolo op.